ckanext/data_qld_theme/templates/scheming/package/read.html

```diff
@@ -6,17 +6,3 @@
     <li class="active"><a href="{{ h.url_for(controller='package', action='read', id=pkg.name) }}">{{ pkg.title }}</a></li>
 {% endblock %}
 
-{% block package_notes %}
```

Comment:
which ticket dropped this? is it 24/26 or 27?

Reply:
It was dropped via [DQL2-26]